lib/components/opaque_image_view.js

```diff
@@ -16,10 +16,18 @@ const GeminiHost = 'd7hftxdivxxvm.cloudfront.net';
 const ImageQuality = 85;
 
 export default class OpaqueImageView extends React.Component {
+  state: {
+    aspectRatio: ?number,
+    width?: number | string,
+    height?: number | string
+  };
```

Comment:
Orly? I see.

How about we just change that to the below, should make the type info much clearer as it’s close to where it’s needed.

```js
const width = this.state.width.toString();
const height = this.state.height.toString();
return `https://${GeminiHost}/?resize_to=${type}&width=${width}&height=${height}&quality=${ImageQuality}&src=${encodeURIComponent(imageURL)}`;
```